eng/pipelines/templates/variables/image.yml

```diff
@@ -17,7 +17,7 @@ variables:
   - name: LINUXNEXTVMIMAGE
     value: azsdk-pool-mms-ubuntu-2204-1espt
   - name: WINDOWSVMIMAGE
-    value: 1espt-win-2022-cpp-rollback
+    value: azsdk-pool-mms-win-2022-1espt
```

Comment:
After this merges can I remove this image?

Reply:
Yes. The image behind it was cleaned up by 1ES